docs/references/nextjs/custom-signup-signin-pages.mdx

```diff
@@ -62,6 +62,22 @@ If Clerk's prebuilt components don't meet your specific needs or if you require
     ```
   </CodeBlockTabs>
 
+  ### Add custom routes to middleware
+
+  `clerkMiddleware()` does not make routes public by default, so we need to make them public for people to access the custom routes to sign in and sign up. In your `middleware.ts` file, update the following code to allow the `sign-in` and `sign-up` pages to be public.
```

Comment:
clerkMiddleware() does make routes public by default. 
let me know if you want to call on this or pair with roy on it - the concept for this section isn't totally correct

Reply:
The key concept, as I mentioned to @kduprey just now in DMs, is that `authMiddleware()` makes the sign-in and a sign-up routes public. It had to do this as it would protect everything by default. My linear comment was a little unclear.

`clerkMiddleware()` makes everything public and does not explicitly make sign-in and sign-up routes public. This can lead to customers who will make all or most routes private and in doing so make their sign-in and sign-up routes private. The linear is just about calling this potential foot gun out and showing how to avoid it.

We can do a call, but I think after DMs with Kenton we're aligned.